content/blog/2024-04-23-april-project-meeting-update/contents.lr

```diff
@@ -0,0 +1,23 @@
+title: Prism: InvenioRDM Partner Meeting Summary, March 2024
```

Comment:
```suggestion
title: InvenioRDM Partner Meeting Summary, March 2024
```

(probably a leftover from another blog post)

Reply:
Yes! Thank you, good catch